apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceContainsRegexAssertionsSpec.kt

```diff
@@ -37,26 +32,30 @@ class CharSequenceContainsRegexAssertionsSpec : Spek({
 
         private fun getNameContainsRegex() = "$contains with search mode $regex"
 
-        private fun getAtLeastTripleString(): Pair<(String, String) -> String, StringFun> =
+        private fun getAtLeastTripleString() =
             { what: String, times: String -> "$contains $what $atLeast $times" } to
-                ("$contains.$atLeast.$regex" to Companion::containsAtLeast)
+                ("$contains.$atLeast.$regex" to Companion::containsAtLeastString)
 
-        private fun getAtLeastTripleRegex(): Pair<(Regex, String) -> String, RegexFun> =
-            { what: Regex, times: String -> "$contains $what $atLeast $times" } to
-                ("$contains.$atLeast.$regex" to Companion::containsAtLeast)
+        private fun getAtLeastTripleRegex() =
+            { what: String, times: String -> "$contains $what $atLeast $times" } to
+                ("$contains.$atLeast.$regex" to Companion::containsAtLeastRegex)
 
-        private fun containsAtLeast(expect: Expect<CharSequence>, atLeast: Int, a: String, aX: Array<out String>) =
-            expect.contains.atLeast(atLeast).regex(a, *aX)
+        private fun containsAtLeastString(
+            expect: Expect<CharSequence>,
+            atLeast: Int,
+            a: String,
+            aX: Array<out String>
+        ) = expect.contains.atLeast(atLeast).regex(a, *aX)
 
-        private fun containsAtLeast(expect: Expect<CharSequence>, atLeast: Int, a: Regex, aX: Array<out Regex>) =
-            expect.contains.atLeast(atLeast).regex(a, *aX)
+        private fun containsAtLeastRegex(expect: Expect<CharSequence>, atLeast: Int, a: String, aX: Array<out String>) =
+            expect.contains.atLeast(atLeast).regex(a.toRegex(), *aX.map { it.toRegex() }.toTypedArray())
 
         private fun getAtLeastIgnoringCaseTripleString() =
             { what: String, times: String -> "$contains $ignoringCase $what $atLeast $times" } to
                 ("$contains.$atLeast.$ignoringCase.$regex" to Companion::containsAtLeastIgnoringCase)
 
         private fun getAtLeastIgnoringCaseTripleRegex() =
```

Comment:
remove this function, is the same as getAtLeastIgnoringCaseTripleString

Reply:
You removed the wrong function, keep containsAtLeastRegex and remove getAtLeastIgnoringCaseTripleRegex instead